hbase-server/src/test/java/org/apache/hadoop/hbase/regionserver/TestHRegionFileSystem.java

```diff
@@ -352,33 +351,36 @@ public void setWorkingDirectory(Path arg0) {
   }
 
   @Test
+  // TODO: move this to test layer of SFT interface
```

Comment:
What is the plan for this? To do for this issue, or a follow up JIRA? Is it filed yet or will it be? etc.

Reply:
Fixing as part next patch on same PR